dao/src/integ-test/java/uk/gov/moj/sdt/dao/BulkSubmissionDaoTest.java

```diff
@@ -168,9 +172,11 @@ public void testGetBulkSubmissionBySdtBulkRef() {
     @Test
     public void testGetBulkSubmissionBySdtBulkRefNotFound() {
         final String sbr = "NO_SUCH_ID";
-        final IBulkSubmission submission =
-                bulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, sbr, dataRetentionPeriod);
-
+        IBulkSubmission submission = null;
+        try {
+            submission = bulkSubmissionDao.getBulkSubmissionBySdtRef(bulkCustomer, sbr, dataRetentionPeriod);
+        } catch (Exception e) {
+        }
```

Comment:
Need to log the exception message here along with a message indicating where the failure occurred.  This will aid debugging if the test fails.

Reply:
if the test fails anyway we can see the log, we don't need to log it, removed this exception block now